src/main/java/org/molgenis/vcf/report/mapper/HtsJdkToPersonsMapper.java

```diff
@@ -8,14 +8,18 @@
 import java.util.Map;
 import org.molgenis.vcf.report.model.Items;
 import org.molgenis.vcf.report.model.Sample;
+import org.phenopackets.schema.v1.core.Pedigree.Person;
 import org.springframework.stereotype.Component;
 
 /**
  * @see VCFHeader
- * @see Sample
+ * @see Person
  */
 @Component
-public class HtsJdkToSamplesMapper {
+public class HtsJdkToPersonsMapper {
```

Comment:
I prefer the original name ```HtsJdkToSamplesMapper``` since it is mapping to ```Sample``` and a revert of the class doc changes.